README.md

```diff
@@ -1 +1,64 @@
-# nvim-package-specification
\ No newline at end of file
+# `plugin.json`
+
+The neovim package specification supports a single, top-level package metadata file. This file can be *either* 'plugin.lua' or 'plugin.json'. The file is a superset of the Rockspec File Format and can contain the following fields
+
+* `package` : String, the name of the package
+
+* `version` : String, the version of the package. Should obey semantic versioning conventions, for example `0.1.0`. For all version identifiers, implementation should check for a `version` prefixed with `v` in the git repository, as this is a common convention.
+
+* `specification_version` : String, the current specification version. (0.1.0) at this time.
```

Comment:
Preferably this could be something shorter, just like jsonrpc does it with just `"jsonrpc": "2.0"`. I don't have any ideas at the moment though

Reply:
maybe `spec`, `specification`? Not sure,

Reply:
`plugspec`?

Reply:
I feel like `plug` has too close of a connotation to `vim-plug`

Reply:
`pluginspec`?

rockspec has `rockspec_format`, we can have `pluginspec_format`.

And if we do that, shell we rename `plugin.lua`/`plugin.json` to `pluginspec.lua`/`pluginspec.json`?